src/functions/uploadToRSIS/application/data-mapper/data-mapper.ts

```diff
@@ -526,20 +528,24 @@ export const formatQuestionCompletedB = (testData: TestData | undefined): Boolea
   return 0;
 };
 
-export const formatQuestionCompleted = (testData: TestData | undefined, questionCount: number): BooleanAsNumber => {
-  let totalFaults: number = 0;
-  const tellMeFaults: QuestionResult[] = get(testData, 'vehicleChecks.tellMeQuestions', null);
-  const showMeFaults: QuestionResult[] = get(testData, 'vehicleChecks.showMeQuestions', null);
+export const formatQuestionCompleted =
+  (testData: TestData | undefined, questionCount: number, delegatedTest?: boolean): BooleanAsNumber => {
```

Comment:
should `delegatedTest` have a default value if it is optional?

Reply:
ignore me :)

Reply:
never 🥇